detection-rules/sender_new_from_domain_first_time_sender.yml

```
@@ -0,0 +1,21 @@
+name: "Sender: new domain (<7d) from first-time sender"
+description: |
+  Detects inbound emails where the sender domain is less than 7 days old from first-time sender 
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and beta.whois(sender.email.domain).days_old <= 7
+  and (
+            (
+                sender.email.domain.root_domain in $free_email_providers
+                and sender.email.email not in $sender_emails
+            )
+            or (
+                sender.email.domain.root_domain not in $free_email_providers
+                and sender.email.domain.domain not in $sender_domains
+            )
+  )
+tags:
+  - "Suspicious sender"
+  - "Domain age"
```

```suggestion
  - "New domain"
```

i think this would be more informative - eg when you're seeing the Tags of rules that fired on a message in the MDV. wdyt?

---

I dig it! I was going to just add Whois enrichment but I like this better